### PR TITLE
Refactor, extract instancePool, translator

### PR DESCRIPTION
**Pull request by panslava**

### CODE REVIEW

File: pkg/l4netlb/l4netlbcontroller_test.go [hunk 1]
```diff
@@ -217,7 +216,6 @@ func newL4NetLBServiceController() *L4NetLBController {
 	}
 
 	lc := NewL4NetLBController(ctx, stopCh)
-	lc.Init()
```
Comment (kl52752) [inline pkg/l4netlb/l4netlbcontroller_test.go:220]: you can just `return NewL4NetLBController(ctx, stopCh)`

Comment (panslava) [reply pkg/l4netlb/l4netlbcontroller_test.go:220]: thank you, fixed!

**Pull request merged**